subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/unixStartScript.txt

```diff
@@ -243,6 +243,14 @@ set -- \\
 <% } %>        ${mainClassName} \\
         "\$@"
 
+# Stop when "xargs" is not available.
+
+command -v xargs > /dev/null 2>&1
+if [ $? -ne 0 ]
```

Comment:
Why not do this:
```suggestion
if ! command -v xargs >/dev/null 2>&1
```
I'm pretty sure this is `sh`-compatible, shellcheck didn't complain.